src/main/java/com/github/fppt/jedismock/operations/sortedsets/AbstractZRange.java

```diff
@@ -116,6 +114,18 @@ protected Slice getSliceFromRange(NavigableSet<ZSetEntry> entries) {
         return Response.array(list);
     }
 
+    protected static Stream<Slice> getSliceStream(ZSetEntry e) {
+        int value = e.getScore().intValue();
```

Comment:
`Math.round` returns the closest `long` to its argument and can be used here instead of boxing. But in order to define if the value is equal to an integer, there are better ways...